spec/controllers/organizations_controller_spec.rb

```diff
@@ -2,12 +2,55 @@
   let!(:organization) { Fabricate(:organization) }
   let(:member) { Fabricate(:member, organization: organization) }
   let(:user) { member.user }
+  let!(:second_organization) { Fabricate(:organization) }
 
   describe 'GET #index' do
     it 'populates and array of organizations' do
       get :index
 
-      expect(assigns(:organizations)).to eq([organization])
+      expect(assigns(:organizations)).to eq([organization, second_organization])
+    end
+  end
+
+  describe 'GET #index (search)' do
```

Comment:
maybe we can move this as a `context` inside the `describe 'GET #index'` block, wdyt?

Reply:
I followed the same structure as in "offers" but it is true that the "Get #index" of organizations is emptier so it could be fine.

Reply:
Yes, inside the block above, as a new `context` seems a better fit 👌🏼